browser/src/canvas/sections/URLPopUpSection.ts

```diff
@@ -71,6 +71,7 @@ class URLPopUpSection extends HTMLObjectSection {
 		app.LOUtil.setImage(imgCopyBtn, 'lc_copyhyperlinklocation.svg', app.map);
 		imgCopyBtn.setAttribute('width', 18);
 		imgCopyBtn.setAttribute('height', 18);
+		imgCopyBtn.setAttribute('alt', _('Copy link location'));
```

Comment:
string in const variable and reuse 2 times (see now we have it 2 times typed here)